drakcore/drakcore/app.py

```diff
@@ -79,14 +80,15 @@ def graph(task_uid):
 
 @app.route("/status/<task_uid>")
 def status(task_uid):
-    # FIXME race condition?
-    try:
-        obj = minio.stat_object("drakrun", task_uid + "/syscall.log")
-    except NoSuchKey:
-        # TODO consult karton if the task is actually running?
-        return jsonify({"status": "pending"})
+    tasks = rs.keys("karton.task:*")
+
+    for task_key in tasks:
+        task = json.loads(rs.get(task_key))
+
+        if task["root_uid"] == str(task_uid) and task["status"] != "Finished":
```

Comment:
What's the purpose of `str(task_uid)`?. It seems that it's already a `str`

Reply:
true, fixed